src/components/dashboard/Reason.js

```diff
@@ -15,12 +28,48 @@ export type AmountProps = {
   styles: any,
 }
 
+class PaymentCategory {
+  static DigitalServices = 1
+
+  static SocialMedia = 2
+
+  static Product = 3
+
+  static Course = 4
+
+  static Donation = 5
+
+  static Other = 6
+
+  static labelOf(category) {
+    const { DigitalServices, SocialMedia, Product, Course, Donation, Other } = this
+
+    switch (category) {
+      case DigitalServices:
+        return 'Digital Services'
+      case SocialMedia:
+        return 'Social Media Management'
+      case Product:
+        return 'Product'
+      case Course:
+        return 'Course / Private Consultation'
+      case Donation:
+        return 'Donation'
+      case Other:
+        return 'Other'
+      default:
+        return ''
+    }
+  }
+}
+
 const SendReason = (props: AmountProps) => {
   const { screenProps } = props
   const { params } = props.navigation.state
 
   const [screenState, setScreenState] = useScreenState(screenProps)
-  const { reason, ...restState } = screenState
+
+  const { reason, isDisabledNextButton, ...restState } = screenState
```

Comment:
why is this part of screenState
screenState is only used to pass data between navigations
it should be simple useState()

Reply:
this is so if user decides to return from the next screen to this screen his category selection will be "remembered"